Trio/Sources/APS/OpenAPS/OpenAPS.swift

```diff
@@ -534,15 +534,26 @@ final class OpenAPS {
 
         // Check for active Temp Targets and adjust HBT if necessary
         try await context.perform {
-            // Check if a Temp Target is active and if its HBT differs from user preferences
+            // Check if a Temp Target is active
```

Comment:
We do still check if the HBT differs (just differently now) and we technically still adjust it here, if need be.